<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="association">
     <property name="title">
      <string>Association</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="instrument_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Instrument</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="instruments">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>(None)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TARDIS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PXRDIP</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="detector_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Current Detector</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="detectors">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="data">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item alignment="Qt::AlignLeft">
         <widget class="QPushButton" name="load">
          <property name="text">
           <string>Select File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="files_label">
          <property name="text">
           <string>(None Selected)</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Transform Image:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="transforms">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mirror about Vertical</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mirror about Horizontal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Transpose</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rotate 90°</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rotate 180°</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rotate 270°</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_transform">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Add Transform</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="outline">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Detector Outline</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="add_template">
        <property name="text">
         <string>Add Detector Outline</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="trans">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Translate</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">adjust_template</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rotate">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Rotate</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">adjust_template</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDialogButtonBox" name="button_box">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
          <property name="centerButtons">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="save">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Save</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="finalize">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Finalize Instrument</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="completed_label">
          <property name="text">
           <string>Completed Detectors:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="completed_dets">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="complete">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Complete and Reload Instrument</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="adjust_template"/>
 </buttongroups>
</ui>
